libraries/Microsoft.Bot.Expressions/ExpressionFunctions.cs

```diff
@@ -3780,8 +3780,7 @@ private static int GetPropertyCount(dynamic obj)
             lookup.Add("lessOrEquals", lookup[ExpressionType.LessThanOrEqual]);
             lookup.Add("not", lookup[ExpressionType.Not]);
             lookup.Add("or", lookup[ExpressionType.Or]);
-
-            lookup.Add("concat", lookup[ExpressionType.Concat]);
+            lookup.Add("&", lookup[ExpressionType.Concat]);
             return lookup;
```

Comment:
Why was this removed?  OK to have & as an inline alias, but don't we want to support concat as well?